pylint/checkers/utils.py

```diff
@@ -444,6 +452,23 @@ def store_messages(func):
     return store_messages
 
 
+def check_messages(*messages: str) -> Callable[[nodes.NodeNG], None]:
+    """Kept for backwards compatibility, deprecated.
+
+    Use only_required_for instead, which conveys the intent of the decorator much clearer.
+    """
+    # Uncomment the following warning once all 'check_messages' calls have been replaced
```

Comment:
I think we can activate it right now so we see if we migrated everything by seeing the warnings in tests :)

Reply:
If this does not lead to a failed CI pipeline, we can activate it right away. 👍